{
  "id": "MAL-2024-5288",
  "summary": "Malicious code in jxhcfdwvun (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "modified": "2024-06-25T13:36:37Z",
  "published": "2024-06-25T13:36:37Z",
  "database_specific": {
    "malicious-packages-origins": [
      {
        "modified_time": "2024-06-25T13:36:37Z",
        "import_time": "2024-06-28T02:49:17.090810082Z",
        "versions": [
          "0.0.1"
        ],
        "id": "RLMA-2024-04070",
        "source": "reversing-labs",
        "sha256": "ea07c659b86dacd77bf33ce78fb034e32f07253bf97a5f29c76f19cefa8df155"
      }
    ]
  },
  "affected": [
    {
      "package": {
        "name": "jxhcfdwvun",
        "ecosystem": "PyPI",
        "purl": "pkg:pypi/jxhcfdwvun"
      },
      "versions": [
        "0.0.1"
      ],
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/jxhcfdwvun/MAL-2024-5288.json"
      }
    }
  ],
  "schema_version": "1.6.0",
  "credits": [
    {
      "name": "ReversingLabs",
      "contact": [
        "https://www.reversinglabs.com"
      ],
      "type": "FINDER"
    }
  ]
}